### PR TITLE
Create GSoD-proposal.md

### PR DESCRIPTION
**Pull request by vchrombie**
This PR adds the GSoD proposal file to the repository.

### CODE REVIEW

File: GSoD-proposal.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+## Project budget
+
+15000 USD
+
+Budget item | Amount | Running Total | Notes/justifications
+------------|--------|---------------|---------------------
+Technical writer develop and implement new documentation for the projects (GrimoireLab idea) | 5000.00 | 5000.00 | 
+Technical writer develop and implement new documentation for the projects (Website idea) | 5000.00 | 10000.00 | 
+Technical writer develop and implement new documentation for the projects (Augur idea) | 5000.00 | 15000.00 | 
+TOTAL |  | 15000.00 |
```
Comment (GeorgLink) [inline GSoD-proposal.md:57]:
```suggestion
Budget item | Amount | Notes/justifications
------------|--------|---------------------
Technical writer for: GrimoireLab | 5,000.00 |
Technical writer for: Website | 5,000.00 |
Technical writer for: Augur idea | 5,000.00 |
TOTAL |  | USD $15,000.00 |
```

Comment (GeorgLink) [inline GSoD-proposal.md]:
```suggestion
$15,000 USD

Budget item | Amount | Running Total | Notes/justifications
------------|--------|---------------|---------------------
Technical writer develop and implement new documentation for the project: **GrimoireLab** | 5,000 | 5,000.00 | 
Technical writer develop and implement new documentation for the project: **Website** | 5,000 | 10,000.00 | 
Technical writer develop and implement new documentation for the project: **Augur**  | 5,000 | 15,000.00 | 
TOTAL |  | USD $15,000.00 |
```

**Comment by vchrombie** [comment]
@GeorgLink I updated the PR, please review when you have time.

File: GSoD-proposal.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Project budget
+
+$15,000 USD
+
+Budget item | Amount | Running Total | Notes/justifications
+------------|--------|---------------|---------------------
+Technical writer develop and implement new documentation for the project: **GrimoireLab** | 5000.00 | 5000.00 | 
+Technical writer develop and implement new documentation for the project: **Website** | 5000.00 | 10000.00 | 
+Technical writer develop and implement new documentation for the project: **Augur** | 5000.00 | 15000.00 |
```
Comment (GeorgLink) [inline GSoD-proposal.md:60]:
```suggestion
Technical writer develop and implement new documentation for the project: **GrimoireLab** | 5,000.00 | 5,000.00 | 
Technical writer develop and implement new documentation for the project: **Website** | 5,000.00 | 10,000.00 | 
Technical writer develop and implement new documentation for the project: **Augur** | 5,000.00 | 15,000.00 | 
```

**Pull request merged**